src/panels/iframe/ha-panel-iframe.ts

```diff
@@ -41,7 +41,7 @@ class HaPanelIframe extends LitElement {
           )}
           src=${this.panel.config.url}
           sandbox="allow-forms allow-popups allow-pointer-lock allow-same-origin allow-scripts allow-modals allow-downloads"
-          allow="fullscreen"
+          allow=${this.panel.config.allow ?? 'fullscreen'}
```

Comment:
This requires a backend change, please link to the core PR

Reply:
Sorry, not sure what PR you're referring to. Could you clarify?

Reply:
The config of this panel is provided by core, so you will need to update core too for this to work.

Reply:
Thank you! I have tried my luck: https://github.com/home-assistant/core/pull/108230. I very much admit that this is a bit out of my comfort zone.